docs/scu/dev/change-default-branch.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Change default branch
+description: How to change the default branch in Spryker CI
+template: concept-topic-template
+---
+
+To change the *default* branch in Spryker CI, do the following:
+
+1. In [Spryker CI](/docs/scu/dev/spryker-ci.html), go to *Projects*.
```

Comment:
@AlexSlawinski , on another similar pages with instructions inside of the same documentation section "Spryker Code Upgrader" we already use bold style (**). So for me personally it doesn't matter what text formatting we will use, I just think that it makes more sense to use the same styling on all pages inside the same documentation section. 

Example with styling on another pages: 

https://github.com/spryker/spryker-docs/blob/master/docs/scu/dev/onboard-to-spryker-code-upgrader/connect-spryker-code-upgrader-to-a-github-managed-project.md?plain=1